lib/decryption.rb

```diff
@@ -0,0 +1,41 @@
+class Decryption
+  attr_reader :character_set
+
+  def initialize
+    @character_set = (("a".."z").to_a << " ").reverse
```

Comment:
Smart thinking to just reverse the set in order to maintain the same character reassignment logic below!!! Also smart to reverse it here so @character set can be normally called in the rest of the class.